### ISSUE TITLE
When extracting, it will continuously output lyrics

### ISSUE THREAD

**Issue by imsohandsome123** [ohollo/chord-extractor#4]
I think that when extracting, not outputting things should make the process faster (saving I/O time). I added '-idqqq' to the timidity command in converters.py, and the verbose has indeed decreased, but it will still be output Lyrics, what is the reason?

**Comment by Kinyugo** [comment]
In `converters.py` change the subprocess call to: 
```python
    subprocess.run(
        ["timidity", midi_path, "-idqqq", "-Ow", "-o", wav_file],
        stdout=subprocess.DEVNULL,
        stderr=subprocess.DEVNULL
    )
```

**Comment by ohollo** [comment]
Thank you both for addressing this, and apologies it has taken a while to respond. It would be great if we could have a parameter in midi_to_wav to silence the timidity outputs, and for the default to be set to silent. @Kinyugo your solution looks good. Please feel free to submit a PR with that (along with the new parameter), and then you can take proper credit for this solution.

**Comment by Kinyugo** [comment]
@ohollo Thank you for considering my contribution. I am working on a PR. 

Quick question, by changing the stdout and stderr to devnull will the following line still work
https://github.com/ohollo/chord-extractor/blob/982c45272d4972a6235abb70b32df64d75715594/chord_extractor/converters.py#L43

**Comment by ohollo** [comment]
@Kinyugo Good point. No, that line may not still work with stdout to devnull. If you know to hand a more robust way to catch a  non midi file being passed in (without stdout) then you can include in your solution. Otherwise you can delete this if statement, and i will come up with a different exception handler later on.
